src/main/java/edu/oregonstate/cs361/battleship/CivShip.java

```diff
@@ -22,7 +22,7 @@ public void hit(BattleshipModel model, boolean isAI)//Occurs when a civilian shi
         if(isAI)
         {
             model.add_computer_hit(start);
-            model.add_computer_hit(new Coord(start.get_x() + diffX/2, start.get_y() + diffY/2));

```

Comment:
I'm pretty sure that we need these divide by 2's.  Unless you changed something else that I am not seeing.